packages/checkbox/README.md

```diff
@@ -133,32 +143,91 @@ of assets, etc. where the checkboxes need to be noticed.
-        <sp-checkbox emphasized indeterminate invalid>Web component</sp-checkbox>
+        <sp-checkbox emphasized indeterminate invalid>
+            Web component
+        </sp-checkbox>
     </div>
 
     <div style="display: flex; flex-direction: column;">
         <h4 class="spectrum-Heading--subtitle1">Disabled</h4>
         <sp-checkbox emphasized disabled>Web component</sp-checkbox>
         <sp-checkbox emphasized checked disabled>Web component</sp-checkbox>
-        <sp-checkbox emphasized indeterminate disabled>Web component</sp-checkbox>
+        <sp-checkbox emphasized indeterminate disabled>
+            Web component
+        </sp-checkbox>
     </div>
 </div>
 ```
 
-### Handling events
+</sp-tab-panel>
+</sp-tabs>
+
+### States
+
+#### Invalid
+
+The `invalid` attribute indicates that the checkbox's value is invalid. When set, appropriate ARIA attributes will be automatically applied.
+
+```html
+<sp-checkbox invalid>Invalid</sp-checkbox>
+```
+
+#### Disabled
+
+The `disabled` attribute prevents the checkbox from receiving focus or events. The checkbox will appear faded.
+
+```html
+<sp-checkbox disabled>Disabled</sp-checkbox>
+```
+
+#### Indeterminate
+
+The `indeterminate` attribute sets the checkbox to an indeterminate state, visually distinct from both checked and unchecked states.
+
+```html
+<sp-checkbox indeterminate>Indeterminate</sp-checkbox>
+```
+
+### Behaviors
+
+#### Handling events
 
 Event handlers for clicks and other user actions can be registered on an `<sp-checkbox>` as they would a standard `<input type="checkbox">` element.
 
 ```html
 <sp-checkbox
     id="checkbox-example"
-    onclick="spAlert(this, '<sp-radio> clicked!')"
+    onclick="spAlert(this, '<sp-checkbox> clicked!')"
 >
-    Web component
+    Check this box to se an onclick alert.
 </sp-checkbox>
 ```
 
-## Accessibility
+### Accessibility
 
 Checkboxes are accessible by default, rendered in HTML using the `<input type="checkbox">` element. When the checkbox is set as `indeterminate` or
 `invalid`, the appropriate ARIA state attribute will automatically be applied.
+
+#### Include a label
+
+Every checkbox must have a label that clearly describes its purpose. The label can be provided as content within the `<sp-checkbox>` element.
+
+```html
+<sp-checkbox>Send me text messages.</sp-checkbox>
+```
+
+#### Label groups of related checkboxes
+
+Sets of checkboxes should always have a clear label that describes what the list of options represents and guides users what to do. This is important for accessibility, since a screen reader will read the label before each option.
+
+```html
+<sp-field-group label="Select your toppings">
+    <sp-checkbox>Ketchup</sp-checkbox>
+    <sp-checkbox>Mustard</sp-checkbox>
+    <sp-checkbox>Pickles</sp-checkbox>
+</sp-field-group>
+```
+
+#### Keyboard Navigation
+
+Checkboxes can be toggled using the <kbd>Space</kbd> key when focused. They follow the standard tab order of the page.
```

Comment:
Can we also surface a section how the screen readers will behave? 

Screen readers interpret checkboxes by announcing their role, label, and current state to the user. This allows users relying on assistive technology to understand and interact with the checkbox effectively.

 - When focused, a screen reader will announce:
     - The role: "checkbox"
     - The label (text provided inside the <sp-checkbox> or associated with it)
     - The state: "checked", "not checked", or "partially checked" (when indeterminate is set)
     - If the checkbox is marked as invalid, it may also announce "invalid entry" depending on the screen reader.